packages/eas-cli/README.md

```diff
@@ -58,8 +59,6 @@ ALIASES
   $ eas login
 ```
 
-_See code: [build/commands/account/login.ts](https://github.com/expo/eas-cli/blob/v0.1.0-alpha.3/build/commands/account/login.ts)_
-
```

Comment:
These deletions related to build commands were generated when I ran `yarn release`.

They are unrelated to my PR and I assume are leftover from a PR that did not follow the `yarn release` procedure.